### DISCUSSION TITLE
Cannot run pwsh file

### DISCUSSION THREAD

**Discussion by veklyyev** [community/community#75305] [Actions] [answered]
### Select Topic Area

Question

### Body

I tried already many variants, it's always failing. I cannot trigger create_directory.ps1 script. Getting an error 'The term './create_directory.ps1' is not recognized as a name of a cmdlet, function, script file, or executable program. '

jobs:
  build-and-install-playwright:
    runs-on: windows-latest

    steps:
      - name: Create Playwright Directory test
        shell: pwsh
        run: |
            ${{env.GITHUB_WORKSPACE}}\create_directory.ps1

**Comment by b8zeek** [comment]
This error message indicates that the PowerShell script you're trying to run cannot be found. 

Try changing the command to this:
```
run: |
  & "${{ env.GITHUB_WORKSPACE }}/create_directory.ps1"
```

Check the file location, see if `create_directory.ps1` script is located in the root of your repository. If it's in a subdirectory, you'll need to include that in the path. Also, check the file permissions: Make sure that the `create_directory.ps1` script has the correct permissions. It should be executable.

Good luck. 🤞🏼 

🍀

**Reply by veklyyev** [reply]
Hi @bejzik8 , thank you for your reply. I did try your solution,  my steps included:

- make sure `create_directory.ps1` is in the root
- provide `create_directory.ps1`executable rights (double-checked)
- change the code line to provided one

And still facing the same error:
```Run & "/create_directory.ps1"
&: D:\a\_temp\0861100f-cd02-[4](https://github.com/veklyyev/wishlist-api/actions/runs/6493481102/job/17634660265#step:2:5)c76-818a-0313ebc91444.ps1:2
Line |
   2 |  & "/create_directory.ps1"
     |    ~~~~~~~~~~~~~~~~~~~~~~~
     | The term '/create_directory.ps1' is not recognized as a name of a cmdlet, function, script file, or executable
     | program. Check the spelling of the name, or if a path was included, verify that the path is correct and try
     | again.

Error: Process completed with exit code 1.```

**Reply by b8zeek** [reply]
Is the repository public?

This error message definitely indicates that the `create_directory.ps1` script cannot be found. Check the path, the script is not in the current directory when command runs.

🍀

**Reply by veklyyev** [reply]
Just in case I put it both into root and  workflow directory. still same behavior. Repository is not public. I can share it with you though.
<img width="237" alt="image" src="https://github.com/community/community/assets/8299735/864115f4-6cd8-46e6-95b9-8a8064f5ccf6">

**Reply by b8zeek** [reply]
Hey, I missed this message; sorry. Were you able to solve the issue?

🍀

**Reply by veklyyev** [reply, marked as answer]
It's fine. I solved it by inlining the file contents. I didn't manage to make it work with the file. Thanks!

**Reply by spunkslut1471** [reply]
llowing access with my biometric keys .. some one blocks me from using or trying to change or access accounts . all started this morning.
this device im using now isnt mune its my brothers. my whole tech world has gone haywire. now my hey google doesnt know anything. about me my devices. network. even my lights ae change colour regularly . smart watch is continually pairing. i cant control anything. now @ 01.02 26/11/2023

**Reply by JjjjAlbrg** [reply]
###

**Comment by JjjjAlbrg** [comment]
steps:
  - name: Create Playwright Directory test
    shell: pwsh
    run: |
        ${{env.GITHUB_WORKSPACE}}\create_directory.ps1

**Comment by spunkslut1471** [comment]
***@***.***
